lib/multipartform.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
+
+function File(path, filename, contentType) {
+  this.buffer = fs.readFileSync(path);;
```

Comment:
Double semicolon here

```suggestion
  this.buffer = fs.readFileSync(path);
```